tests/test_script_variant.py

```diff
@@ -8,13 +8,11 @@
 
 class ScriptVariantTests(unittest.TestCase):
     def test_romanize_japanese_name_by_name_dict(self):
-        assert script_variant.romanize_japanese_name_by_name_dict(
-            u'雪歩') == [u'YUKIHO']
+        results = script_variant.romanize_japanese_name_by_name_dict(u'天海')
+        assert len(results) > 1
```

Comment:
Can you also test that results contain any of the expected romanizations e.g., "TENKAI"?